.github/workflows/link-check.json

```
@@ -0,0 +1,70 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^http(s)?://localhost"
+    },
+    {
+      "pattern": "^http(s)?://127.0.0.1"
+    },
+    {
+      "comment": "This is a private repos",
+      "pattern": "^http(s)?://gitlab.com/ConsenSys/"
+    },
+    {
+      "comment": "This is a private repos",
+      "pattern": "^http(s)?://github.com/ConsenSys/orchestrate"
+    },
+    {
+      "comment": "This is a private repo",
+      "pattern": "^http(s)?://github.com/INFURA/docs(?:/(issues|pulls))?"
+    },
+    {
+      "comment": "Consensys.net now has a ddos attack protection",
```

I am not sure this matters, ( it probably doesn't)  but just updating caps:

```suggestion
      "comment": "Consensys.net now has DDoS attack protection",
```